erpnext/docs/user/manual/en/manufacturing/bill-of-materials.md

```diff
@@ -31,6 +31,11 @@ To make a new BOM:
 
 <img class="screenshot" alt="Update Cost" src="{{docs_base_url}}/assets/img/manufacturing/bom-update-cost.png">
 
+* User can select the currency in the BOM 
+* System calculates the costing based on the price list currency
+
+<img class="screenshot" alt="Update Cost" src="{{docs_base_url}}/assets/img/manufacturing/price-list-based-currency-bom.png">
+
```

Comment:
Please put only the browser page in the screenshot, not the dock and the browser tabs!